PolyEngine/Engine/Src/TransformComponent.cpp

```diff
@@ -0,0 +1,32 @@
+#include "TransformComponent.hpp"
+
+using namespace Poly;
+
+//------------------------------------------------------------------------------
+void TransformComponent::SetLocalTranslation(Vector position) {
```

Comment:
Argument should be `const Vector& translation`, always pass non trivial method arguments as const references instead of values.